### PR TITLE
Updated to Boost 1.67.0

### PR DESCRIPTION
**Pull request by palmerc**

### CODE REVIEW

File: libs/date_time.cmake [hunk 1]
```diff
@@ -8,69 +8,3 @@ _add_boost_lib(
     BOOST_DATE_TIME_STATIC_LINK
     DATE_TIME_INLINE
 )
-
-_add_boost_test(
```
Comment (Orphis) [inline libs/date_time.cmake]: Why would you remove those?

File: libs/filesystem.cmake [hunk 2]
```diff
@@ -14,24 +14,3 @@ _add_boost_lib(
   LINK
     Boost::system
 )
-
-_add_boost_test(
```
Comment (Orphis) [inline libs/filesystem.cmake]: And those?

**Comment by palmerc** [comment]
That is an artifact of an earlier commit. I can rebase.

**Comment by palmerc** [comment]
Fixed. My first pass fix involved bypassing the tests on the 3 libraries I needed and that ended up in my commit.

**Comment by Orphis** [comment]
That looks much better! I'll upload the boost 1.67 archive and use that instead. A bit busy this weekend but I'll get to that next week! :)

**Comment by Orphis** [comment]
I slightly edited the change to remove a VERY long test and use the official Boost distribution and pushed it! Thanks a lot!

**Pull request closed without merging**